CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
 
+## [0.6.4] - 2016-07-27
+### Changed
+- Vanagon now uses one defaut output path for all platforms, with an option
```

Comment:
Should this change warrant a bump to 0.7.0? I'm worried about projects that have a less strict vanagon requirement